src/components/VideoProvider/useLocalTracks/useLocalTracks.test.tsx

```diff
@@ -196,12 +214,12 @@ describe('the useLocalTracks hook', () => {
 
   describe('the removeLocalVideoTrack function', () => {
     it('should call videoTrack.stop() and remove the videoTrack from state', async () => {
-      const { result, waitForNextUpdate } = renderHook(useLocalTracks);
+      const { result, waitForValueToChange } = renderHook(useLocalTracks);
 
       // First, get tracks
       await act(async () => {
         result.current.getAudioAndVideoTracks();
-        await waitForNextUpdate();
+        await waitForValueToChange(() => result.current.localTracks.length);
```

Comment:
i'll need to remember `waitForValueToChange()`, i haven't seen that before!

Reply:
Yeah it's nice! I haven't used it much before. There's also this

`waitFor(() => result.current.localTracks.length > 0);`

It will wait until the condition is `true`. It only exists in a newer major version of the testing library. I tried upgrading, but it broke things. We can upgrade it at another time.